lib/matplotlib/patches.py

```diff
@@ -628,33 +628,50 @@ class Shadow(Patch):
     def __str__(self):
         return "Shadow(%s)" % (str(self.patch))
 
+    @cbook._delete_parameter("3.3", "props")
     @docstring.dedent_interpd
     def __init__(self, patch, ox, oy, props=None, **kwargs):
         """
-        Create a shadow of the given *patch* offset by *ox*, *oy*.
-        *props*, if not *None*, is a patch property update dictionary.
-        If *None*, the shadow will have have the same color as the face,
+        Create a shadow of the given *patch*.
+
+        By default, the shadow will have the same face color as the *patch*,
         but darkened.
 
-        Valid keyword arguments are:
+        Parameters
+        ----------
+        patch : `.Patch`
+            The patch to create the shadow for.
+        ox, oy : float
+            The shift of the shadow in points.
```

Comment:
The shift is in data coordinates, scaled by a factor of dpi/72. 
This might be a mistake after all and arguably it could be better to have it in points, but that would require some more changes below. Possibly there is also an application for "shadows" in data units?

Reply:
I don't understand "data coordinates" vs. "data units". To me they are the same; and mean the the shift is proportional to the axis scale.

But that is not the case. Changing xlim does not change the amount of shift in pixels:

![grafik](https://user-images.githubusercontent.com/2836374/71780370-2fc7b700-2fc2-11ea-88f2-1b52fa08260d.png)
vs.
![grafik](https://user-images.githubusercontent.com/2836374/71780374-4110c380-2fc2-11ea-99ca-0db7f6fea95a.png)

Reply:
Sorry, "data coordinates" and "data units" are synonyms in this context. (After some discussion on this I tried to not use "units" anymore, but was apparently only half successful in doing so. ;-))

I assume the example pictures you show use a patch and shadow within an offset box, with the box transform being the identity? In that case, "data coordinates" would be pixels.  
Maybe instead consider 

```
import matplotlib.pyplot as plt
from matplotlib.patches import Rectangle, Shadow

fig, ax = plt.subplots()
p = Rectangle((0,0), 1, 1)
ax.add_patch(p)
s = Shadow(p, .1, -.1)
ax.add_patch(s)
ax.autoscale()
ax.margins(0.2)

plt.show()
```

Reply:
Thanks for the clarification! Yes my example was using an offset box (simplified version of https://matplotlib.org/devdocs/gallery/text_labels_and_annotations/demo_text_path.html).

Scaled by 72/dpi is not really a nice unit, in particular since we default to 100dpi nowadays. But that's how the API is right now. So, just documenting it.